manifest-editor/src/apps/Splash/Splash.tsx

```diff
@@ -1,27 +1,54 @@
 import { FlexContainerColumn } from "../../components/layout/FlexContainer";
 import { useShell } from "../../context/ShellContext/ShellContext";
 import { RecentFiles } from "../../components/widgets/RecentFiles";
-import { NewTemplates } from "../../components/widgets/NewTemplates";
 import { PaddingComponentLarge } from "../../atoms/PaddingComponent";
+import { LoadManifest } from "../../components/widgets/LoadManifest/LoadManifest";
+import { Layout } from "../../shell/Layout/Layout";
+import { LayoutPanel } from "../../shell/Layout/Layout.types";
+import { ManifestEditorProvider } from "../ManifestEditor/ManifestEditor.context";
 
-export const Splash: React.FC<{ welcome: any }> = ({ welcome }) => {
+export function Splash() {
   const shellContext = useShell();
 
   const handleClick = (id: string) => {
     shellContext.changeResourceID(id);
     shellContext.changeSelectedApplication("ManifestEditor");
   };
 
-  return (
-    <FlexContainerColumn justify="flex-start" style={{ maxWidth: "80%", margin: "auto" }}>
-      <div dangerouslySetInnerHTML={{ __html: welcome }} />
-      <PaddingComponentLarge />
-      <NewTemplates newTemplates={shellContext.newTemplates} changeManifest={handleClick} />
-      <PaddingComponentLarge />
+  const leftPanels: LayoutPanel[] = [];
+
+  const centerPanels: LayoutPanel[] = [
+    {
+      id: "splash",
+      label: "Splash Screen",
+      icon: "",
+      render: () => (
+        <FlexContainerColumn style={{ height: "100%", justifyContent: "space-between" }}>
+          <div>
+            <PaddingComponentLarge />
+            <LoadManifest />
+            <PaddingComponentLarge />
+          </div>
+          <div style={{ width: "100%", height: "50%", backgroundColor: "white" }}>
+            <FlexContainerColumn style={{ width: "90%", margin: "auto" }}>
+              <RecentFiles recentManifests={shellContext.recentManifests} changeManifest={handleClick} />
+            </FlexContainerColumn>
+          </div>
+        </FlexContainerColumn>
+      ),
+    },
+  ];
 
-      <RecentFiles recentManifests={shellContext.recentManifests} changeManifest={handleClick} />
+  const rightPanels: LayoutPanel[] = [];
```

Comment:
We could be moved outside of the `function Splash() {}` and let them be completely static. (Will also make it easier to migrate to the storybook-style later)

Reply:
Ah I missed that, makes sense!

Reply:
Me too the first time I tried it then couldn't work out why it was broken, the build passed but the whole app doesn't render if you break the rules of hooks as the show splash screen logic is one of the first parts of the app it hits!